addons/mail/models/mail_activity.py

```diff
@@ -670,6 +680,14 @@ def _default_activity_type(self):
         ('today', 'Today'),
         ('planned', 'Planned')], string='Activity State',
         compute='_compute_activity_state',
+        compute_raw="""
```

Comment:
Is this a WIP ? Because I feel like this is really not standard and counter productive to have several computation means.

Reply:
Yes, it's POC. There is no specification yet